socorro/processor/signature_utilities.py

```diff
@@ -697,6 +698,40 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         )
         return result
 
+class SignatureShutdownTimeout(SignatureGenerationRule):
+    """Customize the signature for shutdown timeouts"""
+
+    def version(self):
+        return '1.0'
+
+    def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
+        return 'AsyncShutdownTimeout' in raw_crash
```

Comment:
Shouldn't that be `raw_crash.get('AsyncShutdownTimeout')` to also capture if it's there but empty?